pkg/hostagent/api/server/server.go

```diff
@@ -50,6 +50,36 @@ func (b *Backend) GetInfo(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(m)
 }
 
+// DriverConfig is the handler for GET /v1/driver/config and PATCH /v1/driver/config.
+func (b *Backend) DriverConfig(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+	var config interface{}
+	if r.Method == http.MethodPatch {
+		defer r.Body.Close()
+		if err := json.NewDecoder(r.Body).Decode(&config); err != nil {
+			b.onError(w, err, http.StatusBadRequest)
+			return
+		}
+	}
+	config, err := b.Agent.DriverRuntimeConfig(ctx, config)
+	if err != nil {
+		b.onError(w, err, http.StatusInternalServerError)
+		return
+	}
+	m, err := json.Marshal(config)
+	if err != nil {
+		b.onError(w, err, http.StatusInternalServerError)
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	_, _ = w.Write(m)
+}
+
 func AddRoutes(r *http.ServeMux, b *Backend) {
 	r.Handle("/v1/info", http.HandlerFunc(b.GetInfo))
+	r.Handle("GET /v1/driver/config", http.HandlerFunc(b.DriverConfig))
+	r.Handle("PATCH /v1/driver/config", http.HandlerFunc(b.DriverConfig))
```

Comment:
The name of the new handler is not consistent with existing GetInfo. GetConfig and PatchConfig would be consistent.